packages/mobile/src/screens/product-info/components/image-carousel/image-carousel.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FC } from 'react';
+import { ImageSourcePropType, ListRenderItem } from 'react-native';
+import { images } from '~/assets/images/images';
+import { Image, Text, View, FlatList } from '~/components/components';
+import { globalStyles } from '~/styles/styles';
+import { styles } from './styles';
+
+type ItemImageType = {
+  img: ImageSourcePropType;
+  id: number;
+};
+
+const mockData: ItemImageType[] = [
+  { img: images.puff, id: 1 },
+  { img: images.puff, id: 2 },
+  { img: images.puff, id: 3 },
+];
+
+const renderItem: ListRenderItem<ItemImageType> = ({ item, index }) => (
+  <View style={styles.imgContainer}>
+    <Image source={item.img} />
```

Comment:
Looks fine, however, you should count image sizes, otherwise for some screens we won't see card fully

Reply:
fix, add width and height